sudokuSolver/sudokuSolver/sudokuSolverFunctions.cpp

```diff
@@ -0,0 +1,174 @@
+#include "sudokuSolverFunctions.h"
+
+int findMissing(int a[], int b[], int n, int m)
+{
+    for (int i = 0; i < n; i++)
+    {
+        int j;
+        int counter=0;
+        for (j = 0; j < m; j++)
+        {
+            if (a[i] == b[j])
+            {
+                counter++;
+            }
+        }
+        if(counter==0)
+        {
+            return a[i];
+        }
+    }
+}
+
+bool zeroValidator(int a[])
+{
+    for (int i=0; i<9; i++)
+    {
+        if(a[i]==0)
+        {
+            return true;
+        }
+    }
+    return false;
+}
+
+
+void removeDuplicates(std::vector<int> &v)
+{
+    auto end = v.end();
+    for (auto it = v.begin(); it != end; ++it)
+    {
+        end = std::remove(it + 1, end, *it);
+    }
+
+    v.erase(end, v.end());
+}
+
+int findInSquare(int puzzle[9][9],int pozX,int pozY)
+{
+    int listaBrojeva[10]= {0,1,2,3,4,5,6,7,8,9};
```

Comment:
use English names along the whole project

Reply:
listaBrojeva is replaces with arrayOfNumbers